crates/aptos-faucet/src/mint.rs

```diff
@@ -215,7 +215,7 @@ async fn sequences(service: &Service, receiver: AccountAddress) -> Result<(u64,
         .map(|account| account.inner().sequence_number);
     let faucet_seq_num = responses
         .remove(0)
-        .map_err(|_| anyhow::format_err!("faucet account {} not found", faucet_address))?
+        .map_err(|e| anyhow::format_err!("Faucet account {} not found: {:#}", faucet_address, e))?
```

Comment:
This should just log the error, rather than return it to users.

It's telling internals about the faucet

Reply:
I disagree, users who hit this error will just report it directly to us, in which case everyone involved will want as much information as possible.

Reply:
considering we should have our logs, it does seem issue with faucet account is related to our internals - and independent from the actual query from the person reporting the problem. 

on the other hand, there is no faucet in mainnet, so leaking things here is not a huge problem.

leaving to the two of you to resolve, otherwise PR looks good.